.buildkite/pipeline.yml

```diff
@@ -24,7 +24,7 @@ steps:
     agents:
       queue: default
 
-  - label: ":radioactive_sign: Danger - PR Check"
+  - label: ":danger: Danger - PR Check"
```

Comment:
This step has an explicit `notify:` so changing the label text won't impact GitHub Required Checks 👍